feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle-dropdown/configurator-attribute-single-selection-bundle-dropdown.component.html

```diff
@@ -0,0 +1,30 @@
+<div
+  id="{{ createAttributeIdForConfigurator(attribute) }}"
+  class="form-group"
+  *ngIf="attribute?.values?.length !== 0"
```

Comment:
```suggestion
  *ngIf="attribute?.values?.length"
```

`0` is a falsy value, so we don't need to check for `!== 0`